Resources/doc/index.rst

```diff
@@ -164,10 +165,10 @@ ready, use the form component as usual. Then, after you verify the form is valid
     ;
 
     if ($this->getRequest()->getMethod() === 'POST') {
-        $form->bind($this->getRequest());
+        $form->bindRequest($this->getRequest());
 
         if ($form->isValid()) {
-            $em = $this->getDoctrine()->getManager();
+            $em = $this->getDoctrine()->getEntityManager();
```

Comment:
please revert these changes as they are wrong. You are switching to the old Symfony API